src/server/scripts/Northrend/UtgardeKeep/UtgardePinnacle/boss_svala.cpp

```diff
@@ -364,6 +368,11 @@ class boss_svala : public CreatureScript
                                 DoCast(sacrificeTarget, SPELL_RITUAL_PREPARATION);
                                 SetCombatMovement(false);
                                 DoCast(me, SPELL_RITUAL_OF_THE_SWORD);
+                                me->SetControlled(true,UNIT_STATE_ROOT);
+                                me->SetDisableGravity(true);
+                                me->SetCanFly(true);
+                                me->SetHover(true);
```

Comment:
looks wrong to me
never saw a creature with can fly, disable gravity and hover at same time.

Reply:
SetHover was my addition to make her wings flap, the other two were in the previous commit that Killyana diff'd (https://github.com/TrinityCore/TrinityCore/issues/23610#issuecomment-529606023)

Thinking back, I think the thing that made her wings flap was not SetHover, but the flags from L375.  I'll remove and see if the visual is still there.

Reply:
you checked with sniffs what should happens?

Reply:
Resolved with 2a02067

Reply:
@Keader I was unaware that we had sniffs for this boss.  I certainly don't have access to them (and not sure exactly what to do with them if I had them :fearful: )